server/src/weaverbird/backends/mongo_translator/steps/join.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from typing import Any, List
+
+from weaverbird.backends.mongo_translator.utils import column_to_user_variable
+from weaverbird.pipeline import Pipeline, steps
+from weaverbird.pipeline.steps import DomainStep, JoinStep
+
+logger = logging.getLogger(__name__)
+
+
+def translate_join(step: JoinStep) -> List:
+    mongo_pipeline: List[Any] = []
```

Comment:
could it be a list of dictionary ?
```suggestion
    mongo_pipeline: List[Dict] = []
```

Reply:
:+1: